blobs.go

```diff
@@ -103,6 +103,9 @@ type BlobDeleter interface {
 // BlobEnumerator enables iterating over blobs from storage
 type BlobEnumerator interface {
 	Enumerate(ctx context.Context, ingester func(dgst digest.Digest) error) error
+
+	// ModTime returns the modification time of the blob from the storage.
+	ModTime(ctx context.Context, dgst digest.Digest) (time.Time, error)
```

Comment:
@dmage This operation has absolutely nothing to do with enumeration. It is a property of the blob. I think we define a separate blob info struct for backend metadata, as opposed to immutable descriptor data to match. If it is indeed cacheable, like the creation time, then we should add it to redis.